### PR TITLE
feat: Implement RegistrationForm with Formik

### PR DESCRIPTION
**Pull request by olexandor**
- Implemented Formik and Yup for RegistrationForm
- New TextInput wrapper made (TextField) that has a label and error messaging part of it

### CODE REVIEW

**Comment by Cauldrath** [comment]
Could you make a story.tsx file for this component?

**Comment by Cauldrath** [comment]
Also: conflicts.

**Comment by olexandor** [comment]
@Cauldrath Conflicts resolved. Registration form component was added at storybook.

File: yarn.lock [hunk 1]
```diff
@@ -6957,20 +6938,6 @@ form-data@~2.3.2:
...
-  dependencies:
-    deepmerge "^2.1.1"
-    hoist-non-react-statics "^3.3.0"
-    lodash "^4.17.14"
-    lodash-es "^4.17.14"
-    react-fast-compare "^2.0.1"
-    scheduler "^0.18.0"
-    tiny-warning "^1.0.2"
-    tslib "^1.10.0"
-
```
Comment (Cauldrath) [inline yarn.lock]: Why are you removing formik from the yarn.lock? When I run yarn, it looks like it's reversing all your changes in this file.

**Comment by olexandor** [comment]
@Cauldrath yarn.lock file was updated. Please check.

**Pull request merged**